mm2src/coins/eth/v2_activation.rs

```diff
@@ -102,6 +102,32 @@ impl EthCoin {
             Some(d) => d as u8,
         };
 
+        let mut web3_instances = vec![];
+        for node in self.web3_instances.clone() {
+            let mut node = node;
+            let mut transport = node.web3.transport().clone();
+
+            let gui_auth = transport.gui_auth_validation_generator.map(|mut g| {
+                g.coin_ticker = ticker.clone();
+                g
+            });
+
+            transport.gui_auth_validation_generator = gui_auth;
+            let web3 = Web3::new(transport);
+
+            node.web3 = web3;
+            web3_instances.push(node);
+        }
+
+        let mut transport = self.web3.transport().clone();
+        let gui_auth = transport.gui_auth_validation_generator.map(|mut g| {
+            g.coin_ticker = ticker.clone();
+            g
+        });
+
+        transport.gui_auth_validation_generator = gui_auth;
+        let web3 = Web3::new(transport);
```

Comment:
It can be shorter a bit:
```rust
        let mut transport = self.web3.transport().clone();
        if let Some(auth) = &mut transport.gui_auth_validation_generator {
            auth.coin_ticker = ticker.clone();
        }
        let web3 = Web3::new(transport);
```